lib/web_ui/lib/src/engine/compositor/canvas_kit_canvas.dart

```diff
@@ -39,11 +38,11 @@ class CanvasKitCanvas implements ui.Canvas {
   }
 
   void _saveLayerWithoutBounds(ui.Paint paint) {
-    _canvas.saveLayerWithoutBounds(paint);
+    _canvas.saveLayerWithoutBounds(paint as SkPaint);
```

Comment:
Have you looked at dart2js with -O4 on code generated for these ?

Reply:
No, but I thought this was purely a syntactic change. We already had these downcasts. They were just implicit.

@sigmundch @vsmenon Is there a difference between implicit and explicit downcasts as far as `dart2js` is concerned?

Reply:
Actually, changed it to `unsafeCast` suggested by @matanlurey. I checked that every `as` I replaced resulted in a slight drop in code size in `--release` mode.